transports/http3-quiche/src/http3wtstreamvisitor.cc

```diff
@@ -142,8 +154,10 @@ namespace quic
         if (send_fin_)
         {
             bool success = stream_->SendFin();
-            if (success)
+            if (success) {
                 fin_was_sent_ = true;
+                getJS()->processStreamNetworkFinish(NetworkTask::streamFinal);
+            }
```

Comment:
in what cases will `success == false` ? should we do anything with that case?

Reply:
We will retry the next round, as the network was probably blocked.
All other failure cases have separate message handler, that inform js land.